Source/Testably.Abstractions.Testing/FileSystem/FileSystemWatcherMock.cs

```diff
@@ -353,17 +353,27 @@ private bool MatchesFilter(ChangeDescription changeDescription)
 			if (!changeDescription.Path.StartsWith(fullPath,
 				_fileSystem.Execute.StringComparisonMode))
 			{
-				return changeDescription.ChangeType == WatcherChangeTypes.Renamed &&
-				       changeDescription.OldPath?.StartsWith(fullPath,
-					       _fileSystem.Execute.StringComparisonMode) == true;
+				if (!(changeDescription.ChangeType == WatcherChangeTypes.Renamed &&
+				      changeDescription.OldPath?.StartsWith(fullPath,
```

Comment:
```suggestion
				if (!changeDescription.ChangeType == WatcherChangeTypes.Renamed ||
				      !changeDescription.OldPath?.StartsWith(fullPath,
```

I try to avoid negated braces in if-conditions, as I had to read this multiple times ("why don't you check that the old path is **not** in the watched directory?") before understanding it...